devdocumentation.html

```diff
@@ -74,7 +74,7 @@ <h1 class="logo">
     <div class="doc-wrapper">
         <div class="container">
             <div id="doc-header" class="doc-header text-center">
-                <h1 class="doc-title"><i class="icon fa fa-code"></i> Developer Documentation</h1>
+                <h1 class="doc-title"><i class="icon fa fa-code"></i>Developer Documentation</h1>
```

Comment:
The space was wanted for design :)
I'll re-add it.

Reply:
That is strange but why not :)

Reply:
This is to have a little space between the icon and the title.